docs/reference/release-notes/8.13.1.asciidoc

```diff
@@ -3,6 +3,13 @@
 
 Also see <<breaking-changes-8.13,Breaking changes in 8.13>>.
 
+[[known-issues-8.13.1]]
+[float]
+=== Known issues
+
+* Due to a bug in the bundled JDK 22 nodes might crash abruptly under high memory pressure.
+We recommend https://www.elastic.co/guide/en/elasticsearch/reference/current/install-elasticsearch.html#jvm-version[downgrading to JDK 21.0.2] asap to mitigate the issue.
```

Comment:
```suggestion
We recommend <<jvm-version,downgrading to JDK 21.0.2>> asap to mitigate the issue.
```
(Linking by ID will avoid problems in the future)

Reply:
Thanks 👍